engines/private/lex.yy.cpp

```diff
@@ -0,0 +1,1938 @@
+#line 1 "engines/private/lex.yy.cpp"
```

Comment:
I suppose this file needs to be removed.

Reply:
Done.